### PR TITLE
[ImageView] Load cropped images that take screen pixel ratio into account.

### LINKED ISSUE #13: [Image] Get images sized according to screen DPI
E.g. if we want to display an image at `200x100`, we should make GraphQL queries like so:

Screen scale:
1. `200x100`
2. `400x200`
3. `600x300`

For those images where we dynamically size based on the aspect ratio (e.g. masonry grid) we only specify 1 dimension (e.g. width) that’s specified according to screen scale and let metaphysics figure out the resize arguments based on the image’s aspect ratio.

### PR DESCRIPTION
**Pull request by alloy**
Fixes #13

### CODE REVIEW

File: lib/components/artist/shows/medium_image_show.js [hunk 1]
```diff
@@ -11,37 +11,35 @@ import OpaqueImageView from '../../opaque_image_view'
...
-
   fragments: {
     show: () => Relay.QL`
       fragment on PartnerShow {
         href
         meta_image {
-          cropped(width: $imageWidth, height: $imageHeight) {
-            url
-          }
+          url(version: "large")
```
Comment (mennenia) [inline lib/components/artist/shows/medium_image_show.js]: so we're using the large image url everywhere, and just sizing the view accordingly for large, medium, and small images, right?

**Comment by mennenia** [comment]
:+1:

File: lib/components/artist/shows/medium_image_show.js (hunk 1 above)
Comment (alloy) [inline lib/components/artist/shows/medium_image_show.js]: Yeah, we’re using the URL for the large version to ask [Gemini](https://github.com/artsy/gemini) for a scaled/cropped version at a specific size. This is the same as metaphysics does when asking for a specific size, but we don’t know the exact sizes we need ahead of time so can’t really do that from Relay.

Comment (mennenia) [inline lib/components/artist/shows/medium_image_show.js]: Awesome. That's what I gathered, just wanted to verify ^_^.

**Comment by alloy** [comment]
Rebased and good to merge 👍

**Pull request merged**